src/pudl/etl.py

```diff
@@ -92,33 +98,25 @@ def _validate_params_eia(etl_params):  # noqa: C901
     if not eia_input_dict['eia860_years'] and eia_input_dict['eia923_years']:
         eia_input_dict['eia860_years'] = eia_input_dict['eia923_years']
 
-    # Validate the etl_params
-    if eia_input_dict['eia860_tables']:
-        for table in eia_input_dict['eia860_tables']:
-            if table not in pc.pudl_tables["eia860"]:
-                raise AssertionError(
-                    f"Unrecognized EIA 860 table: {table}"
-                )
-
-    if eia_input_dict['eia923_tables']:
-        for table in eia_input_dict['eia923_tables']:
-            if table not in pc.pudl_tables["eia923"]:
-                raise AssertionError(
-                    f"Unrecogized EIA 923 table: {table}"
-                )
-
-    for year in eia_input_dict['eia860_years']:
-        if year not in pc.working_years['eia860']:
-            raise AssertionError(f"Unrecognized EIA 860 year: {year}")
-
-    for year in eia_input_dict['eia923_years']:
-        if year not in pc.working_years['eia923']:
-            raise AssertionError(f"Unrecognized EIA 923 year: {year}")
-    if (not eia_input_dict['eia923_years']
-            and not eia_input_dict['eia860_years']):
-        return None
-    else:
-        return eia_input_dict
+    if (eia_input_dict['eia860_ytd']
+        & (pd.to_datetime(pc.working_years['eia860m'][0]).year
+           in eia_input_dict['eia860_years'])):
+        raise AssertionError(
+            "Attempting to integrate an eia860m year "
+            f"({pd.to_datetime(pc.working_years['eia860m'][0]).year}) that is "
```

Comment:
I'd really like the user to never be able to change the year_month for 860m, which is why I didn't immediately want to add it to the settings file. Not all months are available for every year. Really we only ever want to pull in the most recent month in the year after the last available 860 year.

I suppose we could always determine this via the available year_month combos in the datastore instead of looking at constants. Or convert the `eia860_ytd` to `eia860_year_month`.